.github/workflows/rebase.yml

```diff
@@ -0,0 +1,20 @@
+name: Automatic Rebase
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  rebase:
+    name: Rebase
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
```

Comment:
Shouldn't `token: ${{ secrets.GITHUB_TOKEN }}` be here too, based on examples on https://github.com/cirrus-actions/rebase ?

Reply:
I agree, I don't think the action can push on the branch without the secret 🤔